### PR TITLE
UI?

### PR DESCRIPTION
**Pull request by JaninaHeikkala**
please

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -2,187 +2,62 @@
 	// NOTE: Might want to move assets outside of /src/ since that would build it differently
 	// NOTE: Since these are static images we have to import them like this, since the svelte compiler apparently does not
 	// allow for things like <img src="/bla/blorg/blirg.png">
-	import img from "$lib/images/strommingPog.png"
-	import vid from "$lib/videos/video.mp4"
-
-	import { Grid, gridHelp } from "$lib/js/svelte-grid";
+	
+	import Header from "./Header.svelte";
```
Comment (JoarSabel) [inline src/routes/+page.svelte]: Import never used? Remove?

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -2,187 +2,62 @@
...
+	import InfoTab from "./tabs/InfoTab.svelte";
+	import VideoPlayerTab from "./tabs/VideoPlayerTab.svelte";
+	import { fade } from 'svelte/transition';
```
Comment (JoarSabel) [inline src/routes/+page.svelte]: same as above

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -2,187 +2,62 @@
...
+	import Tabs from "./tabs/Tabs.svelte";
 
 	const id = () => "_" + Math.random().toString(36).substr(2, 9);
```
Comment (JoarSabel) [inline src/routes/+page.svelte]: same as above?

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -2,187 +2,62 @@
...
+	STRØMMING
+</section>
 
-	<button on:click={add} class="bg-blue-500 rounded p-2 text-white">
-		add
-	</button>
+<!--<svelte:head>
+	<title></title>
+	<meta name="description" content="Svelte demo app" />
+</svelte:head>-->
```
Comment (JoarSabel) [inline src/routes/+page.svelte]: Might as well remove this comment section

File: src/routes/+page.svelte [hunk 5]
```diff
@@ -2,187 +2,62 @@
...
-		</video> 
-</section>
 
-<style>
 
-	.remove { cursor: pointer; position: absolute; right: 5px; top: 3px; }
+<style>
 
-	section {
+	main {
```
Comment (JoarSabel) [inline src/routes/+page.svelte]: unused css class?

File: src/routes/+page.svelte [hunk 6]
```diff
@@ -2,187 +2,62 @@
...
-		border-color: white !important;
+	section {
+		font-size: 60px;
+		text-align: center;
 	}
 
-	:global(.svlt-grid-item) {
-		background: red;
-	}
+	.remove { cursor: pointer; position: absolute; right: 5px; top: 3px; }
```
Comment (JoarSabel) [inline src/routes/+page.svelte]: unused?

File: src/routes/styles.css [hunk 7]
```diff
@@ -17,17 +17,18 @@
 }
 
 body {
-	min-height: 100vh;
+	/*min-height: 100vh;
```
Comment (JoarSabel) [inline src/routes/styles.css]: remove comment

File: src/routes/styles.css [hunk 8]
```diff
@@ -17,17 +17,18 @@
...
 	margin: 0;
-	background-attachment: fixed;
-	background-color: var(--color-bg-1);
-	background-size: 100vw 100vh;
-	background-image: radial-gradient(
+	/*background-attachment: fixed;
+	background-color: var(--color-bg-1);*/
+	/*background-size: cover;*/
```
Comment (JoarSabel) [inline src/routes/styles.css]: remove comment

File: src/routes/styles.css [hunk 9]
```diff
@@ -17,17 +17,18 @@
...
+	background: url(../lib/images/background.png) no-repeat center center fixed;
+	/*background-image: radial-gradient(
```
Comment (JoarSabel) [inline src/routes/styles.css]: remove comment

File: src/routes/tabs/InfoTab.svelte [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+      border-right: 1px solid #dee2e6;
+      border-top: 1px solid #dee2e6;
+    }
+    p {
+      font-size: 17px;
+      line-height: 30px;
+      margin-bottom: 30px;
+    }
+    
+    .split {
```
Comment (JoarSabel) [inline src/routes/tabs/InfoTab.svelte]: unused?

File: src/routes/tabs/InfoTab.svelte [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+    }
+    img {
```
Comment (JoarSabel) [inline src/routes/tabs/InfoTab.svelte]: unused?

File: src/routes/tabs/InfoTab.svelte [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+      border-radius: 1rem;
+    }
+    blockquote {
```
Comment (JoarSabel) [inline src/routes/tabs/InfoTab.svelte]: unused?

File: src/routes/tabs/InfoTab.svelte [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+        font-size: 30px;
+        color: #888;
+        position: relative;
+    }
+    
+    blockquote:before {content: "";position: absolute;left: -40px;top: 17%;height: 60px;width: 4px;background: #6878d7;}
```
Comment (JoarSabel) [inline src/routes/tabs/InfoTab.svelte]: unused?

File: src/routes/tabs/VideoPlayerTab.svelte [hunk 14]
```diff
@@ -0,0 +1,187 @@
...
+
+    let items = [
+    {
+		6: gridHelp.item({
+			x: 0,
+			y: 0,
+			w: 4,
+			h: 5,
+		}),
+		//id: id(),
```
Comment (JoarSabel) [inline src/routes/tabs/VideoPlayerTab.svelte]: nitpick: remove comment

File: src/routes/tabs/VideoPlayerTab.svelte [hunk 15]
```diff
@@ -0,0 +1,187 @@
...
+
+	function add() {
+		let newItem = {
+			6: gridHelp.item({
+			w: 2,
+			h: 2,
+			x: 0,
+			y: 0,
+			}),
+			id: id(),
```
Comment (JoarSabel) [inline src/routes/tabs/VideoPlayerTab.svelte:66]: I think this might be my error from earlier? Or is it new?

**Review by JoarSabel — CHANGES_REQUESTED** [review]
Mostly nitpicking of the code

File: src/routes/tabs/VideoPlayerTab.svelte (hunk 15 above)
Comment (JaninaHeikkala) [reply src/routes/tabs/VideoPlayerTab.svelte:66]: i copied your code to the tab :)

File: src/routes/+page.svelte (hunk 3 above)
Comment (JaninaHeikkala) [reply src/routes/+page.svelte]: idk if this should be in the videoplayer tab mby?

File: src/routes/tabs/VideoPlayerTab.svelte (hunk 15 above)
Comment (JaninaHeikkala) [reply src/routes/tabs/VideoPlayerTab.svelte:66]: i moved the id that was in page.svelte here now. should be fine

File: src/routes/+page.svelte (hunk 3 above)
Comment (JoarSabel) [reply src/routes/+page.svelte]: leave for now

**Review by JoarSabel — APPROVED** [review]

**Comment by JoarSabel** [comment]
ready for Mörj

**Pull request merged**